src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java

```diff
@@ -188,18 +165,23 @@ private static String generateHeader(String pluginName){
      *
      * @return String  total documentation for the page
      */
-    public static String generatePluginHelp(String pluginName, Map<String, List<StepDescriptor>> byPlugin, boolean genHeader){
+    public static String generatePluginHelp(String pluginName, String displayName, Map<String, List<StepDescriptor>> byPlugin, boolean genHeader){
         Main.isUnitTest = true;
 
         //TODO: if condition
         StringBuilder whole9yards = new StringBuilder();
         if(genHeader){
-            whole9yards.append(generateHeader(pluginName));
+            whole9yards.append(generateHeader(displayName));
```

Comment:
I like the idea of linking to the plugins.jenkins.io page.  It'd be awesome to link to other content like tutorials; however that's more difficult since it'd require storing that information somewhere.